snippets/base/views.py

```diff
@@ -114,11 +116,14 @@ def fetch_render_snippets(request, **kwargs):
                          .select_related('template')
                          .filter_by_available())
 
+    current_firefox_version = (
+        version_list(product_details.firefox_history_major_releases)[0].split('.', 1)[0])
```

Comment:
nit: You could also use the `LooseVersion/StrictVersion` from `distutils.version` if you feel that helps the readability:

```python
v = max(map(StrictVersion, product_details.firefox_history_major_releases.keys()))
v.version
```